src/api/admin.ts

```diff
@@ -14,13 +17,16 @@ app.post('/sslCerts', async (req, res) => {
     success: true,
     message: 'SSL Certs and domain name records successfully created'
   }
+
   try {
-    const sslCertResponse = await createSslCerts(
-      serviceResponse,
-      service,
-      selectedDomain
-    )
-    if (!sslCertResponse.success) return res.json(sslCertResponse)
+    if(ENV === 'production'){
+      const sslCertResponse = await createSslCerts(
+        serviceResponse,
+        service,
+        selectedDomain
+      )
+      if (!sslCertResponse.success) return res.json(sslCertResponse)
+    }
 
     const cnameResponse = await setCnameRecords(
```

Comment:
could you also add a check here.... to only setCnameRecords if we are in production?